src/main/kotlin/com/autonomousapps/internal/models.kt

```diff
@@ -416,3 +417,29 @@ internal data class ConsoleReport(
     }
   }
 }
+
+data class AbiExclusions(
```

Comment:
should be `internal`.

Reply:
I did at first but it was a bit more invasive than I expected due to being used in constructor params later. I'll do it in an isolated commit and can revert if we want

Reply:
Not as bad as I thought actually 354351c